.github/CONTRIBUTING.md

```diff
@@ -122,28 +122,17 @@ Even though bundled in this repo, the website is regarded as a separate project.
 
 ### Local previews
 
-It is recommended to exclude `./website/public/` from your editor if you want efficient searches.
-
-To install the required node modules, type:
-
-```bash
-npm install && cd website && npm install && cd ..
-```
-
-For local previews on http://localhost:4000, type:
-
-```bash
-npm run web:start # that gets you just the website. if you need companion, etc. you can use `npm start` instead
-```
-
-This will watch the website, as well as Uppy, as the examples, and rebuild everything and refresh your browser as files change.
+1. `npm install`
+2. `cd website && npm install && cd ..` (even if you ran `npm run bootstrap` before)
```

Comment:
after a root npm install you need to npm run bootstrap again, so this could read

```
1. `npm install`
1. `npm run bootstrap`
1. `cd website && npm install && cd -`
...
```

Reply:
@goto-bus-stop, great, thank you!